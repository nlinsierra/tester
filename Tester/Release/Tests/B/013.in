2C 8S 9S AD 4S 2D 2S KC 3S JD JH AS 3D 2H QS 6S 4D 7S KD QD 6D TH 6C 3C 3H KS 9H TC 9D AC 8H 7D 5S
