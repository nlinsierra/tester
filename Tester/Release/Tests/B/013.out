19
4C 5C 7C 8C 9C JC QC 5D 8D TD 4H 5H 6H 7H QH KH AH TS JS
